test/distributed/org/apache/cassandra/fuzz/sai/SingleNodeSAITestBase.java

```diff
@@ -95,52 +97,60 @@ public static void before() throws Throwable
         sut = new InJvmSut(cluster);
     }
 
-    public SingleNodeSAITestBase(boolean withAccord)
+    public SingleNodeSAITestBase(TransactionalMode transactionalMode)
     {
-        this.withAccord = withAccord;
+        this.transactionalMode = transactionalMode;
     }
 
     @Test
     public void basicSaiTest()
+    {
```

Comment:
FYI @ifesdjeen is rewriting this to be a random schema.  So there is conflict and feel that your patch will likely go in first... but just wanted to point out this pending conflict

Reply:
I did talk with @ifesdjeen  and he took in some of my changes already on his branch so we can merge in either order.